lfc/core/src/main/kotlin/org/lflang/generator/uc/UcCmakeGenerator.kt

```diff
@@ -50,6 +50,7 @@ abstract class UcCmakeGenerator(
             |cmake_minimum_required(VERSION $minCmakeVersion)
             |project(${mainTarget} LANGUAGES C)
             |set(LF_MAIN_TARGET ${mainTarget})
+            |set(SOURCE_FOLDER $S{CMAKE_SOURCE_DIR}/../../../src)
```

Comment:
This makes some assumptions about the project layout. I think instead we should leverage that we have the path to the LF source file available at compile time. I think the include path should be computed based on that path instead